broker-cli/proto/broker.proto

```diff
@@ -589,10 +589,29 @@ message GetBlockOrdersResponse {
 
 /**
  * Request to retrieve all block orders in a given market.
+ * This rpc returns a max record count of 1000 due to transport limits of grpc. Please
```

Comment:
Is the max defined as a default in gRPC? If not, where is this implemented?

Reply:
@bradleystachurski max size message of GRPC is 4mb. I've removed the max record account and added a separate ticket to handle the max message requirement